### ISSUE TITLE
[Bug] Action rolls not working in v10

### ISSUE THREAD

**Issue by AlexBlck** [ElfFriend-DnD/foundryvtt-dnd5eCharacterActions#42]
When trying to roll any action, nothing happens in the game and the following error appears in console:
```
Uncaught TypeError: app._onItemRoll is not a function
    at HTMLDivElement.<anonymous> (modules/character-actions-list-5e/module/foundryvtt-dnd5eCharacterActions.js:339)
    at HTMLDivElement.dispatch (jquery.min.js:2)
    at HTMLDivElement.y.handle (jquery.min.js:2)
```
It's there even with no other modules active.

Foundry version 10.284
DnD5e version 2.0.2

**Comment by akrigline** [comment]
What version of the module are you using? This should be fixed in 6.0.x

**Comment by AlexBlck** [comment]
> What version of the module are you using? This should be fixed in 6.0.x

Hmm, mine was 5.0.2 and claimed to be the latest!
![image](https://user-images.githubusercontent.com/39992844/188891026-7f5074f1-f206-4301-8a0f-7bf726da6e0b.png)
Manually deleting and re-installing fixes this, thanks.

Now the only issue is that I see this when I roll if Monk's Enhanced Journal is active, but I guess that's a different issue.
![image](https://user-images.githubusercontent.com/39992844/188892452-6ef00c38-71a9-4acd-9c4f-7b0e6c9cc704.png)

**Comment by akrigline** [comment]
huh. that 'current' is wierd...
will have to keep an eye out and see if that pops up for anyone else
